data/net/minecraft/world/phys/shapes/Shapes.mapping

```diff
@@ -29,29 +29,61 @@ CLASS net/minecraft/world/phys/shapes/Shapes
 		ARG 4 selectionContext
 		ARG 5 rotationAxis
 		ARG 6 possibleHits
+	METHOD create (DDDDDD)Lnet/minecraft/world/phys/shapes/VoxelShape;
+		ARG 0 x1
+		ARG 2 y1
+		ARG 4 z1
+		ARG 6 x2
+		ARG 8 y2
+		ARG 10 z2
```

Comment:
I think the case should be made for replacing these x1 -> x2, etc with minX -> maxX, etc in multiple places.